rmgpy/rmg/react.py

```diff
@@ -57,14 +59,21 @@ def react(*spcTuples):
     """
 
     from rmgpy.rmg.main import maxproc
-    
-    # Available RAM memory (kB)
-    mem = psutil.virtual_memory()
-    usermem = mem.available
-    
-    tmp = divmod(usermem, resource.getrusage(resource.RUSAGE_SELF).ru_maxrss)
+   
+    # Get available RAM (GB)and procnum dependent on OS
+    if platform == "linux" or platform == "linux2":
+        # linux
+        memoryavailable = psutil.virtual_memory().free / (1000.0 ** 3)
+        memoryuse = psutil.Process(os.getpid()).memory_info()[0]/(1000.0 ** 3)
+    elif platform == "darwin":
+        # OS X
+        memoryavailable = psutil.virtual_memory().available/(1000.0 ** 3)
+        memoryuse = resource.getrusage(resource.RUSAGE_SELF).ru_maxrss/(1000.0 ** 3) 
+
```

Comment:
I think you should add handling for Windows as well. It doesn't have to be equivalent handling, but you could simply add an `else` for any other platform and perhaps only use one processor in that case.

Reply:
Done, but couldn't test it. Is someone with a Windows OS volunteering?